_sass/_header.scss

```diff
@@ -31,6 +32,10 @@
       }
     }
   }
+  
+.navbar.navbar-custom.navbar-fixed-top.top-nav-collapse a.navbar-brand.page-scroll {
```

Comment:
The `a` in front of `.navbar-brand` is usually left out because it’s too specific.